src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -452,6 +452,36 @@ public function testFindAlternativeNamespace()
         }
     }
 
+    public function testFindAlternativesOutput()
+    {
+        $application = new Application();
+
+        $application->add(new \FooCommand());
+        $application->add(new \Foo1Command());
+        $application->add(new \Foo2Command());
+        $application->add(new \Foo3Command());
+
+        $expectedAlternatives = array(
+            'afoobar',
+            'afoobar1',
+            'afoobar2',
+            'foo1:bar',
```

Comment:
I do understand that, my point is that we need to take care of that `:` when sorting like a human would do.